docs/arcadia-users-group/20220906-intro-to-shell1/lesson.md

```diff
@@ -0,0 +1,294 @@
+This is because the command line automatically looks in the current working directory if we don't specify anything else about its location (it's starting from where we are). 
+
+We can also run the same command on the same file using an **absolute path**:
+
+```
+head ~/README.md
+head /home/joyvan/README.md
+```
+
+The previous three commands all point to the same file. 
+But the first way, `head README.md`, will only work if we are entering it while "sitting" in the directory that holds that file, while the second and third ways will work no matter where we happen to be in the computer. 
+
+It is important to always think about *where* we are in the computer when working at the command line. 
+One of the most common errors/easiest mistakes to make is trying to do something to a file that isn't where we think it is. 
+Let's run `head` on the "README.md" file again, and then let's try it on another file: "index.md":
+
+```bash
+head README.md
+head index.md
+```
+
+Here the `head` command works fine on "README.md", but we get an error message when we call it on "index.md" telling us no such file or directory. 
+If we run the `ls` command, we can see the computer is absolutely right – spoiler alert: it usually is – and there is no file here named "index.md". 
+
+Here is how we can run `head` on "index.md" by specifying an accurate **relative path** to that file:
+
+```bash
+head docs/index.md
+```
+
+## Tab-completion
+
+Tab-completion is a huge time-saver, but even more importantly it helps prevent mistakes. 
+
+If we are trying to specify a file that's in our current working directory, we can begin typing its name and then press the <kbd>tab</kbd> key to complete it. 
+If there is only one possible way to finish what we've started typing, it will complete it entirely for us.
+If there is more than one possible way to finish what we've started typing, it will complete as far as it can, and then hitting <kbd>tab</kbd> twice quickly will show all the possible options. 
+If tab-complete does not do either of those things, then the file we're looking for is not in the directory we're pointing to.
+
+## Navigating by the command line
+
+We can also move into the directory containing the file we want to work with by using the `cd` command (**c**hange **d**irectory). 
+This command takes a positional argument that is the path (address) of the directory we want to change into. 
+This can be a relative path or an absolute path. 
+Here we'll use the relative path of the subdirectory, "docs", to change into it (use tab-completion!):
+
+```bash
+cd docs
+pwd
+ls
+head index.md
+```
+
+Great. But now how do we get back "up" to the directory above us? 
+One way would be to provide an absolute path, like `cd /home/joyvan`, but there is also a shortcut.
+`..` are special characters that act as a relative path specifying "up" one level – one directory – from wherever we currently are. 
```

Comment:
Just to show that these can be combined, it may worth doing a demo of what two levels up may look like.